web/template/partial/course/manage/create-lecture-form.gohtml

```diff
@@ -1,127 +1,216 @@
+            <li class="flex items-center " :class="currentTab === 0 ? `text-blue-500 dark:text-white` : `text-gray-400 dark:text-gray-500`">
+                <span class="flex items-center justify-center w-5 h-5 mr-2 text-xs border rounded-full shrink-0" :class="currentTab === 0 ? `border-blue-500 dark:border-white` : `border-gray-400 dark:border-gray-500`">
+                    1
+                </span>
+                <span class="hidden sm:inline-flex sm:ml-2">Lecture Type</span>
+                <svg aria-hidden="true" class="w-4 h-4 ml-2 sm:ml-4" fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M13 5l7 7-7 7M5 5l7 7-7 7"></path></svg>
+            </li>
+            <li class="flex items-center " :class="currentTab === 1 ? `text-blue-500 dark:text-white` : `text-gray-400 dark:text-gray-500`">
+                <span class="flex items-center justify-center w-5 h-5 mr-2 text-xs border rounded-full shrink-0" :class="currentTab === 1 ? `border-blue-500 dark:border-white` : `border-gray-400 dark:border-gray-500`">
+                    2
+                </span>
+                <span class="hidden sm:inline-flex sm:ml-2">Details</span>
+                <svg x-show="formData.vodup" aria-hidden="true" class="w-4 h-4 ml-2 sm:ml-4" fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M13 5l7 7-7 7M5 5l7 7-7 7"></path></svg>
+            </li>
+            <li x-show="formData.vodup" class="flex items-center " :class="currentTab === 2 ? `text-blue-500 dark:text-white` : `text-gray-400 dark:text-gray-500`">
+                <span class="flex items-center justify-center w-5 h-5 mr-2 text-xs border rounded-full shrink-0" :class="currentTab === 2 ? `border-blue-500 dark:border-white` : `border-gray-400 dark:border-gray-500`">
+                    3
+                </span>
+                Video Files
+            </li>
+        </ol>
 
-        <label x-show="!formData.premiere && !formData.vodup && !loading">
-            <span class="text-sm text-5">Lecture Hall</span>
-            <select class="tl-select" id="lectureHallBulkSelect" x-model="formData.lectureHallId">
-                <option class="text-gray-300" value="0">Self streaming
-                </option>
-                {{range .LectureHalls}}
-                    <option value="{{.Model.ID}}" class="text-gray-300">
-                        {{.Name}}
-                    </option>
-                {{end}}
-            </select>
-        </label>
+        <div class="form-container-body">
+            <div x-show="currentTab === -1" x-data="{ progress: { COMB: 0, PRES: null, CAM: null } }" @voduploadprogress.window="(e) => (progress = e.detail)">
```

Comment:
The tabs could potentially be in their own templates if that makes things a little easier to understand